acrobat/styles/styles.css

```diff
@@ -386,3 +387,9 @@ div.how-to ol li::before{
   margin: auto;
   padding: 20px;
 }
+
+@media (max-width: 1200px) {
+  .how-to-heading {
+    padding-top: 52px;
```

Comment:
This will add unnecessary padding on smaller screens. Widget changes height at screen width of 768px, so you can do this: 
```
@media (min-width: 768px) {
  .how-to-heading {
    padding-top: 52px;
  }
```

Reply:
Hi @zagi25 thanks for suggestion! I have updated the code to align with padding values which were used on old design, please take a look. Each device resolution has its own specific values, which is why I had to define additional for mobile/tablet